APCAppCore/APCAppCoreTests/Reminder Tests/APCTasksReminderManagerTests.m

```diff
@@ -0,0 +1,240 @@
+//
+//  APCTasksReminderManagerTests.m
+//  APCAppCore
+//
+//  Created by Michael L DePhillips on 1/26/16.
+//  Copyright © 2016 Apple, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "MockAPCTasksReminderManager.h"
+
+static NSString* const kWalkingTaskReminder = @"WalkingTaskReminder";
+static NSString* const kCardiacTaskReminder = @"CardiacTaskReminder";
+static NSString* const kBalanceTaskReminder = @"BalanceTaskReminder";
+
+static NSString* const kWalkingName = @"Walking Assessment";
+static NSString* const kCardiacName = @"Cardiac Health Activity";
+static NSString* const kBalanceName = @"Balance Assessment";
+
+@interface APCTasksReminderManagerTests : XCTestCase
+@property (nonatomic, strong) MockAPCTasksReminderManager* reminderManager;
+@end
+
+@implementation APCTasksReminderManagerTests
+
+/** Put setup code here. This method is called before the invocation of each test method in the class. */
+- (void) setUp
+{
+    [super setUp];
+    
+    self.reminderManager = [[MockAPCTasksReminderManager alloc] init];
```

Comment:
If you are going to use a property that you define in setup, then you should nil it out in teardown.